tests/test_client.py

```diff
@@ -1429,20 +1430,21 @@ def test_add_attachment_guesses_mimetype(record_setup: Client, tmp_path):
 
     p = tmp_path / "file.txt"
     p.write_text("hello")
```

Comment:
nit: is this still necessary if you mock `open()`?

Reply:
Ah it's not, I cleaned it up.